app/lib/tool/neat_task/pub_dev_tasks.dart

```diff
@@ -27,6 +28,14 @@ void setupFrontendPeriodicTasks() {
     task: backfillNewFields,
   );
 
+  // Checks the Datastore integrity of the model objects.
+  _weekly(
+    name: 'check-datastore-integrity',
+    isRuntimeVersioned: true,
+    task: () async =>
+        await IntegrityChecker(dbService, concurrency: 2).verifyAndLogIssues(),
```

Comment:
What happens in case the integrity check crashes? What effect will it have on the service?

Reply:
Good question, I guess it will bring down the isolate + probably the VM, making that instance to reboot. Luckily there will be some time before it is being retried. I'm planning to add some memory logging before/after the periodic tasks are running, so that we will have some data there.

Reply:
Is there any chance we could move it to another isolate/process?

Reply:
This is already a non-web-serving isolate. We could move this to a non-frontend service, e.g. `analyzer`, that way we don't risk bringing down the API...